Jellyfin.Server.Implementations/Devices/DeviceManager.cs

```diff
@@ -78,23 +100,20 @@ await using (dbContext.ConfigureAwait(false))
                 dbContext.Devices.Add(device);
 
                 await dbContext.SaveChangesAsync().ConfigureAwait(false);
+                var newDevice = await dbContext.Devices
+                    .Include(d => d.User)
+                    .FirstOrDefaultAsync(d => d.Id == device.Id)
+                    .ConfigureAwait(false);
+                _devices.TryAdd(device.Id, newDevice!);
```

Comment:
Since we know an item is added, this can be `.First()` and the null-suppression can be removed